Fakebook.Posts/Fakebook.Posts.UnitTests/Controllers/DeleteCommentTests.cs

```diff
@@ -13,7 +18,13 @@
 using Xunit;
 
 namespace Fakebook.Posts.UnitTests.Controllers {
-    public class DeleteCommentTests {
+    public class DeleteCommentTests : IClassFixture<WebApplicationFactory<Startup>> {
```

Comment:
all this stuff is integration tests, not unit tests, and so they don't belong in the unit test project and shouldn't contribute to code coverage statistics.
they're good though.

Reply:
So it is ok that we leave it for now to count toward the code coverage.

Reply:
yeah